test/runtests_ep.jl

```diff
@@ -0,0 +1,44 @@
+using TSPDrone
+using Test, Statistics
+
+include("../src/tspd_utils.jl")
+include("../src/tsp_ep_all.jl")
```

Comment:
Please comment out these two lines. 

Also include this `runtests_ep.jl` in the `runtests.jl` file.   The GitHub action automatically run `runtests.jl` for error checking.

Reply:
Line 5 is required because by using 'TSPDrone', the unchanged version of 'exact_partitioning' is being run. Without this line, it will lead to an 'unsupported keyword argument' error. This line can be removed after merging.

Reply:
Line 5 is required because by using **TSPDrone**, the unchanged version of **exact_partitioning** is being run. Without this line, it will lead to an 'unsupported keyword argument' error. This line can be removed after merging.

Reply:
Line 5 is required because by using TSPDrone, the unchanged version of exact_partitioning is being run. Without this line, it will lead to an 'unsupported keyword argument' error. This line can be removed after merging.

Reply:
Line 5 is required because by using **TSPDrone**, the unchanged version of **exact_partitioning** is being run. Without this line, it will lead to an 'unsupported keyword argument' error. This line can be removed after merging.

Reply:
Without `line 4`, while running **exact_partitioning**, an error will occur since the function cannot access the **objective_value** function in `tspd_utils.jl`. **exact_partitioning** was never used solely in the original structure; it was always utilized inside **tsp_ep_all**, which is where this new issue emerges.

Reply:
‘using TSPDrone’ will do the job.

Reply:
On your computer try 

] test TSPDrone